update-tests-yaml-files

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash -e
+#
+#  Update or add one or more key:value pair in tests.yaml for all charms in charms.txt
+#  Then, propose gerrit reviews.
+#
+#  e.g. reset_timeout is now default 180s for bundletester
+#  ./update-tests-yaml-files master --kvs reset_timeout:600 will update or add
+#  this value in the first tests.yaml it finds in each charm.
+# 
+#  Note there should only be one tests.yaml per charm.
+#
+
+NO_REVIEW="True"
```

Comment:
I don't see anywhere where NO_REVIEW gets set to anything but True. I may be missing something.